mnefun/_mnefun.py

```diff
@@ -2037,6 +2075,17 @@ def do_preprocessing_combined(p, subjects, run_indices):
                        **old_kwargs)
             raw.add_proj(projs)
             raw.apply_proj()
+            if p.autoreject_thresholds:
+                eog_epochs = create_eog_epochs(raw, tmin=eog_t_lims[0],
+                                               tmax=eog_t_lims[1],
+                                               ch_name=p.ecg_channel)
+                new_dict = get_rejection_threshold(eog_epochs)
```

Comment:
Here and above we use `autoreject` to set the rejection params. We need to verify that it's not eliminating too many EOG epochs. Have you checked to see what proportion are removed? It should be fairly small for most datasets.